api/src/glfs-fops.c

```diff
@@ -6239,9 +6227,6 @@ pub_glfs_xreaddirplus_r(struct glfs_fd *glfd, uint32_t flags,
 
     GF_REF_GET(glfd);
 
-    GF_VALIDATE_OR_GOTO(this->name, xstat_p, out);
-    GF_VALIDATE_OR_GOTO(this->name, res, out);
```

Comment:
This is a public function. We don't control the callers, so argument check is a must.

Reply:
done